docs/ref/contrib/sitemaps.txt

```diff
@@ -384,10 +384,10 @@ with a caching decorator -- you must name your sitemap view and pass
     from django.views.decorators.cache import cache_page
 
     urlpatterns = [
-        url(r'^sitemap\.xml$',
+        path('sitemap.xml',
             cache_page(86400)(sitemaps_views.index),
             {'sitemaps': sitemaps, 'sitemap_url_name': 'sitemaps'}),
-        url(r'^sitemap-(?P<section>.+)\.xml$',
+        path('sitemap-<section>.xml',
```

Comment:
`<path:section>`?

Reply:
Again, I'm going to assume that `.+` *ought* to have been `[^/]+`.